Loops Practice/css/style.css

```diff
@@ -0,0 +1,44 @@
+
+body {
+  background-color: #3f8abf;
+  box-sizing: border-box;
+  color: #fff;
+  font-family: "Gotham Rounded A", "Gotham Rounded B", "Gotham Rounded", "Helvetica Neue", Helvetica, Arial, sans-serif;
+}
+
+main {
+  max-width: 1080px;
+  margin: 0 auto;
+  padding: 60px 0 0;
+  width: 90%;
+}
+
+main div {
+  background-color: white;
+  border-radius: 50%;
+  display: inline-block;
+  color: #3f8abf;
+  font-size: 1.25em;
+  font-weight: 500;
+  height: 55px;
+  line-height: 58px;
+  margin: 0.5em;
+  text-align: center;
+  width: 55px;
+}
+
+h1 {
+  font-size: 3em;
+	line-height: 1.35;
+  text-shadow: 0 2px 1px rgba(0,0,0, 0.15);
+}
+
+h2 {
+  font-size: 2.5em;
+  font-weight: normal;
+}
+
+p {
+  font-size: 2em;
+  line-height: 0.8;
+}
```

Comment:
I think most file types want an empty line as the last line.
That's what this icon is saying.
It doesn't really matter but its always nice to not have any kind of warning indicator 😄 .